install/build.py

```diff
@@ -15,7 +15,7 @@
 
 minimum_cuda_version = 9020
 minimum_cudnn_version = 7600
-maximum_cudnn_version = 8099
+maximum_cudnn_version = 8199
```

Comment:
It's ok to proceed to merge, but I think we can reconsider whether this check is still needed or not.

Reply:
Let me check why the check is/was needed.

Reply:
You mean only for maximum, or both minimum and maximum?

Reply:
I meant maximum only. We don't check max version for CUDA, so I felt it is unnatural to limit cudnn.

Reply:
Makes sense. I will make another PR to remove it!